specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/2017-12-01/examples/PerformanceTiersListByLocation.json

```diff
@@ -9,50 +9,83 @@
       "body": {
         "value": [
           {
-            "id": "Basic",
-            "backupRetentionDays": 7,
+            "Id": "Basic",
+            "MaxBackupRetentionDays": 35,
+            "MaxStorageMB": 1024000,
+            "MinBackupRetentionDays": 7,
+            "MinStorageMB": 5120,
             "serviceLevelObjectives": [
               {
-                "edition": "Basic",
-                "dtu": 50,
-                "storageMB": 51200,
-                "id": "PGSQLB50"
+                "Edition": "Basic",
+                "HardwareGeneration": "Gen4",
+                "Id": "PGSQL_B_Gen4_1",
+                "VCore": 1
               },
               {
-                "edition": "Basic",
-                "dtu": 100,
-                "storageMB": 51200,
-                "id": "PGSQLB100"
+                "Edition": "Basic",
+                "HardwareGeneration": "Gen4",
+                "Id": "PGSQL_B_Gen4_2",
```

Comment:
I did not see the PGSQL_ prefix in our tier id. When I called the GetTiers API, it just returned `B_Gen4_2`

Reply:
I am able to see that when I am running it on C# generated SDK

Reply:
yes that is right, I will change in my example.